src/components/ItemsList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import firebase from '../lib/firebase';
+import { useCollectionData } from 'react-firebase-hooks/firestore';
+
+const ItemsList = () => {
+  const [itemsList, loading, error] = useCollectionData(
+    firebase.firestore().collection('items'),
+    {
+      snapshotListenOptions: { includeMetadataChanges: true },
+    },
```

Comment:
I wonder if we need `snapshotListenOptions` here. Is the query loading customized with it?

Reply:
Thanks @Lukabaramishvili  for the review.  I am not really sure whether the second optional parameter is necessary or not. I am still trying to make sense of how firebase works. Unfortunately, almost all  the other participants are just starting out with `firebase` as well. It appears to understand the [`react-firebase-hooks`](https://github.com/CSFrequency/react-firebase-hooks/tree/1e893b11a41df8618a80ac7964bdf02dcf05735e/firestore) doc., one must be quite knowledgeable in `firebase`. I have seen them use firebase jargon  to explain some of these parameters.  I thought it was "harmless" to leave it there but I am now afraid it might become dead code.  

I will remove it if I test the app and its removal doesn't affect anything. Pardon my ignorance.

Reply:
@nibble0101  No worries at all. I was honestly curious because I'm also using `react-firebase-hooks` for the first time.
Testing the app without `snapshotListenOptions` sounds like a good plan. I also think that if it is not necessary at this stage we can remove it. 💯 Thanks for a quick response.

Reply:
I think we don't have to worry too much about this in this stage. As we progress and learn more about firebase we can always change this. Maybe in the final story where we'll have time to make the app more professional